modeling-cmds/src/ok_response.rs

```diff
@@ -349,6 +349,13 @@ define_ok_modeling_cmd_response_enum! {
             pub entity_ids: Vec<Uuid>,
         }
 
+        /// The response from the `EntityMakeHelix` command.
+        #[derive(Debug, Serialize, Deserialize, JsonSchema, ExecutionPlanValue, ModelingCmdOutput)]
+        pub struct EntityMakeHelix {
+            /// The UUID of the helix that was created.
+            pub helix_id: Uuid,
```

Comment:
Let's follow the pattern from other commands, i.e. the frontend chooses the ID and passes it to the backend. So, this shoudl probably not return anything.